src/Display.js

```diff
@@ -0,0 +1,107 @@
+import { elt } from "./util/dom";
```

Comment:
(Having a display/ dir with all the display-related code is probably a good idea)